index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
+    <script src="js/script.js"></script>
```

Comment:
- [ ] I kindly suggest that you move your javascript tags to before the end of your closing body tag as that will prevent your javascript queries from being loaded first before your HTML page. 
- Currently, I get this error because your JavaScript is loaded before your HTML. Moving your scripts tag to before your closing body tag or by using the [DOMContentloaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) function to wrap your javascript queries

https://user-images.githubusercontent.com/52934128/179555710-d5f5edfc-0529-461f-8467-ab77ea4bba6f.mp4


window.addEventListener('DOMContentLoaded', (event) => {
    console.log('DOM fully loaded and parsed');
});